app/__init__.py

```diff
@@ -162,6 +162,11 @@ def _csp(config):
 
 
 def create_app(application):
+    @application.context_processor
+    def inject_feature_flags():
+        feature_best_practices_enabled = os.getenv("FEATURE_BEST_PRACTICES_ENABLED", "false").lower() == "true"
+        return dict(FEATURE_BEST_PRACTICES_ENABLED=feature_best_practices_enabled)
```

Comment:
Nice!  Is the idea here to build a template context that could hold future feature flags as well?

A couple of things go through my mind with this:

* Should we be pulling from the API to make sure the value of the flag matches?  Having the fallback in the admin app itself makes sense, but just something that crossed my mind!
* Similar to the API, I think we should still add the flag to the app config itself here first, then read from the app config when building this dictionary for the template contexts.

Reply:
Yes! I was trying to think of an easier way to allow access to feature flags without having to import them everywhere in order to use them in a template. While I am new to Jinja, I have used this concept with state management before. 

Are you thinking about in order to enhance accuracy, we fetch the feature values from API instead of configs in the admin project? That is a bit of a different approach then originally taken to allow the easy update to the admin without redeploy ( I am sure you can do the same updates to admin instead, it depends on which project we want to update to toggle features)

I think I understand what you are saying for the last bit. I should use app.config['FEATURE.... instead of calling os.getenv(). 


Let me make those changes!

Reply:
Ah good point, I might be overcomplicating things with the API side of it - if feature flags are living only in the admin side, then no need to make API calls!  That's likely getting ahead of ourselves and overcomplicating things a bit. 🙂